_posts/2022-02-10-where-are-they-now-holly.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Where are they now? Holly
+description: We catch up with ex-competitor Holly.
+---
+_In this new blog series we’ll be catching up with some of Student Robotics ex-competitors to see what they are now up to and how competing in Student Robotics has helped them to get there._
```

Comment:
I think I was originally intending this message to only be on the first blog post of each series.
```suggestion
```